src/main/java/de/tum/in/www1/artemis/web/rest/LtiResource.java

```diff
@@ -133,10 +133,10 @@ public void launch(@ModelAttribute LtiLaunchRequestDTO launchRequest, @PathVaria
 
         log.info("handleLaunchRequest done");
 
-        // If the current user was created within the last 15 seconds, we just created the user
+        // If the current user was created within the last 15 minutes, we just created the user
```

Comment:
Am I missing something, or why does it say minutes now, but below you are still just comparing seconds?

Reply:
I convert to TimeUnit.SECONDS.toMinutes before comparing

Reply:
ah I see, it makes sense then :+1: I only was confused because you only changed the comment, but nothing else related to this